### ISSUE TITLE
failed: ActorModelSpec / DispatcherModelSpec

### ISSUE THREAD

**Issue by patriknw** [akka/akka#26954]
https://jenkins.akka.io:8498/job/akka-release-2.5/23

```
[info] - must process messages in parallel *** FAILED *** (15 milliseconds)
[info]   java.lang.AssertionError: assertion failed: expected: Terminated Actor[akka://ActorModelSpec/user/$0p#1469070805] but got unexpected message Terminated(Actor[akka://ActorModelSpec/user/$1p#1943012613])
[info]   at scala.Predef$.assert(Predef.scala:223)
[info]   at akka.testkit.TestKitBase.expectMsgPF(TestKit.scala:437)
[info]   at akka.testkit.TestKitBase.expectMsgPF$(TestKit.scala:433)
[info]   at akka.testkit.TestKit.expectMsgPF(TestKit.scala:923)
[info]   at akka.testkit.TestKitBase.expectTerminated(TestKit.scala:451)
[info]   at akka.testkit.TestKitBase.expectTerminated$(TestKit.scala:450)
[info]   at akka.testkit.TestKit.expectTerminated(TestKit.scala:923)
[info]   at akka.actor.dispatch.DispatcherModelSpec.$anonfun$new$37(ActorModelSpec.scala:595)

```

**Comment by patriknw** [comment]
https://jenkins.akka.io:8498/job/akka-release-2.5-2.11/13

**Comment by patriknw** [comment]
https://jenkins.akka.io:8498/job/akka-release-2.5/64/consoleText

**Comment by johanandren** [comment]
This was fixed in 517ee4c581d463b65a1c1547079fe49d5ef16209 but not backported, I'll do that.
